service/src/main/java/com/codeforcommunity/processor/ProcessorImpl.java

```diff
@@ -187,16 +193,16 @@ public boolean attendEvent(String eventCode, int userid) {
       result = db.select().from(Tables.EVENTS).where(Tables.EVENTS.CODE.eq(eventCode))
           .fetchSingleInto(EventReturn.class);
     } catch (NoDataFoundException e) {
+      e.printStackTrace();
       return false;
     }
 
-    if (result == null || !result.getOpen() || LocalDateTime.now().compareTo(result.getDate()) >= 0)
+    if (result == null || !result.getOpen() || LocalDateTime.now().compareTo(LocalDateTime.parse(result.getDate())) >= 0)
```

Comment:
You can remove the null check since jOOQ will never give you back a null value. You never want to be checking for things that cannot happen because it just adds confusion

Reply:
Good catch, I'll remove that